prebid-server/endpoints/openrtb2/video.md

```diff
@@ -0,0 +1,355 @@
+      "consentrequired": false,
+      "consentstring": ""
+    }
+  },
+  "device": {
+    "ua": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_6_8) AppleWebKit/537.13 (KHTML, like Gecko) Version/5.1.7 Safari/534.57.2",
+    "ip": "123.145.167.10",
+    "devicetype": 1,
+    "ifa": "AA000DFE74168477C70D291f574D344790E0BB11"
+  },
+  "includebrandcategory": {
+    "primaryadserver": 1
+  },
+  "video": {
+    "w": 640,
+    "h": 480,
+    "mimes": [
+      "video/mp4"
+    ],
+    "protocols": [
+      2,
+      3
+    ]
+  },
+  "content": {
+    "episode": 6,
+    "title": "episodeName",
+    "series": "TvName",
+    "season": "season3",
+    "len": 900,
+    "livestream": 0
+  }
+}
+```
+
+### Post Response
+
+The POST response contains an array of `adPod` objects which represents the `adPods` in the request. Each value of the `adPod ` object (`podid:1`, `podid:2`, etc) contain the key-value targeting for those bids and optionally any errors encountered. 
+
+```javascript
+{
+  "adPods": [
+    {
+      "podid": 1,
+      "targeting": [
+        {
+          "hb_pb": "20.00",
+          "hb_pb_cat_dur": "20.00_413_30s",
+          "hb_cache_id": "420d7329-30e8-4c4e-8eaa-fe937172e4e0"
+        },
+        {
+          "hb_pb": "10.00",
+          "hb_pb_cat_dur": "10.00_395_30s",
+          "hb_cache_id": "420d7329-30e8-4c4e-8eaa-fe937172e4e0"
+        },
+        {
+          "hb_pb": "10.00",
+          "hb_pb_cat_dur": "10.00_401_30s",
+          "hb_cache_id": "420d7329-30e8-4c4e-8eaa-fe937172e4e0"
+        }
+      ]
+    },
+    {
+      "podid": 2,
+      "targeting": [
+        {
+          "hb_pb": "10.00",
+          "hb_pb_cat_dur": "10.00_409_30s",
+          "hb_cache_id": "420d7329-30e8-4c4e-8eaa-fe937172e4e0"
+        },
+        {
+          "hb_pb": "10.00",
+          "hb_pb_cat_dur": "10.00_394_30s",
+          "hb_cache_id": "420d7329-30e8-4c4e-8eaa-fe937172e4e0"
+        },
+        {
+          "hb_pb": "15.00",
+          "hb_pb_cat_dur": "15.00_399_30s",
+          "hb_cache_id": "420d7329-30e8-4c4e-8eaa-fe937172e4e0"
+        }
+      ],
+      "errors": {
+        "openx": [
+          {
+            "code": 1,
+            "message": "The request exceeded the timeout allocated"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+## SSAI processing of the response
+The SSAI should take the key-values from the response `adPods.[].targeting.[]${key}` and pass it to the primary ad server as keywords. Because `adPods` do not have specific targeting, an `adPod` can target any bid as long as the bid duration matches that of the `adPod`. 
+
+
+
+## Error Codes
+
+{: .table .table-bordered .table-striped }
+| Id | Description  | Resolution |
+| --- | --- | --- |
+| 1  | Syntax error. |  Fix syntax in request. |
+| 2  | Required param missing. |  Add missing param(s) to request. |
+| 3  | Config not defined. |  Add missing config into backend database. |
+| 4  | Bidder Timeout. |  Add missing config into backend database. |
```

Comment:
Resolution for Id 4 matches the text used for Id 3

Reply:
Nice catch, fixed.